requirements.txt

```diff
@@ -1,15 +1,76 @@
+appnope==0.1.3
+argon2-cffi==21.3.0
+argon2-cffi-bindings==21.2.0
+asttokens==2.0.8
 attrs==22.1.0
+backcall==0.2.0
+bcrypt==4.0.1
+beautifulsoup4==4.11.1
+bleach==5.0.1
+cffi==1.15.1
 click==8.1.3
-colorama==0.4.6
-exceptiongroup==1.0.0
+debugpy==1.6.3
+decorator==5.1.1
+defusedxml==0.7.1
+entrypoints==0.4
+executing==1.1.0
+fastjsonschema==2.16.2
 Flask==2.2.2
-iniconfig==1.1.1
+Flask-Bcrypt==1.0.1
+Flask-SQLAlchemy==3.0.2
+ipykernel==6.16.0
+ipython==8.5.0
+ipython-genutils==0.2.0
+ipywidgets==8.0.2
 itsdangerous==2.1.2
+jedi==0.18.1
 Jinja2==3.1.2
+jsonschema==4.16.0
+jupyter==1.0.0
+jupyter-console==6.4.4
+jupyter-core==4.11.1
+jupyter_client==7.3.5
+jupyterlab-pygments==0.2.2
+jupyterlab-widgets==3.0.3
 MarkupSafe==2.1.1
+matplotlib-inline==0.1.6
+mistune==2.0.4
+nbclient==0.6.8
+nbconvert==7.1.0
+nbformat==5.6.1
+nest-asyncio==1.5.6
+notebook==6.4.12
 packaging==21.3
-pluggy==1.0.0
+pandocfilters==1.5.0
+parso==0.8.3
+pexpect==4.8.0
+pickleshare==0.7.5
+prometheus-client==0.14.1
+prompt-toolkit==3.0.31
+psutil==5.9.2
+psycopg2==2.9.5
+ptyprocess==0.7.0
+pure-eval==0.2.2
+pycparser==2.21
+Pygments==2.13.0
 pyparsing==3.0.9
-pytest==7.2.0
-tomli==2.0.1
+pyrsistent==0.18.1
+python-dateutil==2.8.2
+python-dotenv==0.21.0
+pytz==2022.4
+pyzmq==24.0.1
+qtconsole==5.3.2
+QtPy==2.2.1
+Send2Trash==1.8.0
+six==1.16.0
+soupsieve==2.3.2.post1
+SQLAlchemy==1.4.44
+stack-data==0.5.1
+terminado==0.16.0
+tinycss2==1.1.1
+tornado==6.2
+traitlets==5.4.0
+wcwidth==0.2.5
+webencodings==0.5.1
 Werkzeug==2.2.2
+widgetsnbextension==4.0.3
```

Comment:
Thanks for freezing the requirements--are all of these requirements from importing `flask_bcrypt`? It looks like a lot! This may be the case, I'm just curious 👍

Reply:
All of these were from SQL Alchemy and flask bycrypt, and also flask-sqlalchemy